TestGame/environment.dme

```diff
@@ -1,8 +1,5 @@
 // BEGIN_INTERNALS
 // END_INTERNALS
-// BEGIN_FILE_DIR
-#define FILE_DIR .
-// END_FILE_DIR
```

Comment:
These lines are generated by the Dream Maker IDE and should be left there so we aren't fighting it.